docs/source/contributing.md

```diff
@@ -1,19 +1,21 @@
-# ✨ Contributing to the SATRE Specification
+(contributing)=
 
-🎉  **Welcome to the SATRE specification!** 🎉
+# {{sparkles}} Contributing to the SATRE Specification
```

Comment:
Is there a reason to not put the emoji in the text files?

It feels more portable, unicode supports it and it will render in your text editor/on GitHub.

Reply:
I was assuming that some people might be using text editors that don't support emoji. Happy to revert though.

Reply:
Removed.